### PR TITLE
Styled swiftype search

### PR DESCRIPTION
**Pull request by gsf**
New search using swiftype but on its own page at /search. 

@karissademi @atogle please review.

### CODE REVIEW

**Comment by karissademi** [comment]
I would like to discuss the results/fields  that are being returned.

File: js/phila-scripts.js [hunk 1]
```diff
@@ -1,12 +1,58 @@
...
+/*! phila.gov-theme 0.16.10 phila-theme.js 2015-10-16 9:51:08 PM */
 new List("filter-list", {
     valueNames: [ "item", "item-desc" ]
 });
 
-function searchPhilaGov() {
-    var input = document.getElementById("search-form"), value = input ? input.value : "defaultText";
-    window.location.href = "https://www.google.com/#q=site:phila.gov+" + escape(value);
-}
+var browse = {
```
Comment (atogle) [inline js/phila-scripts.js:6]: I can't tell what this block is doing without a lot of squinting. :) Let's see if we can clear it up a bit.

File: js/dev/search.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+(function ($) {
```
Comment (atogle) [inline js/dev/search.js:1]: This file looks great. Very clear.

**Comment by atogle** [comment]
Generally looks great. The `js/phila-scripts.js` addition was very cryptic. I'd like to make that more readable before we move forward.

File: js/phila-scripts.js (hunk 1 above)
Comment (gsf) [inline js/phila-scripts.js:6]: The block that was removed?

Comment (gsf) [inline js/phila-scripts.js:6]: Oh, this browse block. That isn't part of the search. Not sure how it sneaked into the PR.

Comment (atogle) [inline js/phila-scripts.js:6]: Awesome. Let's drop it then.

Comment (gsf) [inline js/phila-scripts.js:6]: Looks like it was missed in master. I've committed there and will rebase.

**Comment by gsf** [comment]
We have a card in Trello regarding the phila-scripts.js and how the grunt pipeline could be cleaned up. I'll create an issue on this repo to track it closer to the source.

**Pull request merged**